src/lib/Menu/ContentRightSidebarBuilder.php

```diff
@@ -169,16 +183,36 @@ public function createStructure(array $options): ItemInterface
                         : array_merge($createAttributes, ['disabled' => 'disabled']),
                 ]
             ),
-            self::ITEM__EDIT => $this->createMenuItem(
-                self::ITEM__EDIT,
-                [
-                    'extras' => ['icon' => 'edit'],
-                    'attributes' => $canEdit
-                        ? $editAttributes
-                        : array_merge($editAttributes, ['disabled' => 'disabled']),
-                ]
-            ),
-            self::ITEM__MOVE => $this->createMenuItem(
+        ]);
+
+        if ($contentIsUser) {
```

Comment:
Can you extract it to a method? It's becoming harder and harder to maintain. `addEditMenuItem()` is a good candidate.